obtainium/README.md

```
@@ -6,16 +6,47 @@ config site expect. Setup instructions and channel reference live in
+`FLAVORS` definition, so the links and the import files cannot disagree:
 
 ```bash
-python3 obtainium/generate-links.py
+python3 obtainium/generate-links.py          # regenerate
+python3 obtainium/generate-links.py --check  # verify, exits 1 on drift
 ```
 
+Run `--check` after editing anything in this directory. Editing a generated file
+by hand is always wrong — change `generate-links.py` and regenerate.
+
+### How many entries a user can have
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)